### PR TITLE
MGMT-15007: support multi-node cluster installation

### PR DESCRIPTION
**Pull request by danielerez**
As the number for required nodes are now loaded form config-image, need to change the previous logic for updating hosts' ignition and installer-args (see current prepare-cluster-installation script).

Hence, extracted the logic to update-hosts script and added a service for invoking it on rendevouz host.

### CODE REVIEW

File: data/scripts/bin/update-hosts.sh.template [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -e
+
+source issue_status.sh
+status_issue="90_start-install"
```
Comment (nmagnezi) [inline data/scripts/bin/update-hosts.sh.template:5]: where is this being used?

Comment (danielerez) [reply data/scripts/bin/update-hosts.sh.template]: actually it's redundant:) removed.

**Review by nmagnezi — APPROVED** [review]
/lgtm

**Pull request merged**